bullet-featherstone/src/SDFFeatures.cc

```diff
@@ -1205,7 +1205,24 @@ bool SDFFeatures::AddSdfCollision(
 
       auto *world = this->ReferenceInterface<WorldInfo>(model->world);
 
-      if (isStatic)
+      // Set static filter for collisions in
+      // 1) a static model
+      // 2) a fixed base link
+      // 3) a (non-base) link with zero dofs
+      bool isFixed = false;
+      if (model->body->hasFixedBase())
+      {
+        // check if it's a base link
+        std::size_t linkID = model->body->getUserIndex();
+        isFixed = std::size_t(_linkID) == linkID;
```

Comment:
consider renaming `linkID` so it doesn't appear to be the same variable as `_linkID`. Or
```suggestion
        isFixed = std::size_t(_linkID) ==  model->body->getUserIndex();
```

Reply:
changed to one liner. [8841d70](https://github.com/gazebosim/gz-physics/pull/611/commits/8841d701c452030f1efd712e2353d1b675642cd6)